addons/mail/static/src/discuss/core/public/welcome_page.xml

```diff
@@ -24,6 +24,9 @@
                     <button class="btn fa-stack align-self-end p-0 m-3 rounded-circle fs-1 shadow" t-attf-class="{{ state.videoStream ? 'btn-dark' : 'btn-danger' }}" t-on-click="onClickVideo">
                         <i class="fa fa-camera"/>
                     </button>
+                    <button t-if="state.videoStream"  class="btn fa-stack p-0 m-3 rounded-circle fs-1 shadow" t-attf-class="{{ store.settings.useBlur ? 'btn-dark' : 'btn-danger' }}" t-on-click="onClickBlur" t-att-title="store.settings.useBlur ? 'Remove Blur Background' : 'Blur Background'">
```

Comment:
`t-att-title` use here is not working with translation.
Should either indirect in JS and use `_t()` or the `t-att-title` value should be stored in template ctx as a variable and use property of text node in OWL template that are translated:

```xml
<t t-set="blurTittle"><t t-if="store.settings.useBlur">Remove Blur Background</t><t t-else="">Blur Background</t></t>
```